### DISCUSSION TITLE
Uninstall analyst?

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#7179] [Unsupported Versions] [answered]
I accidentally ran so-analyst-install and don't need it. It doesn't really bother me, but I'm going to assume it eats up a certain amount of resources. Is it possible to uninstall it without reinstalling SO2?

**Comment by dougburks** [comment, marked as answer]
You could take a look at the code in `/usr/sbin/so-analyst-install` to see what packages it installed and then manually uninstall those packages.

**Reply by Sazerak** [reply]
Thanks, I wasn't sure if there was an easy way to determine what was installed. I ended up having to recreate my VM regardless so it wasn't an issue anymore, but decided to leave the question up since I'm sure someone else will have the same question in the future. Cheers!

**Reply by Bagu-Hanto** [reply]
@dougburks I navigated to that path, but it is not showing **so-analyst-install** within **usr/sbin**

![image](https://user-images.githubusercontent.com/74457313/178539245-14ea1ac7-9df4-4964-8df7-9067a21644a1.png)

**Reply by TOoSmOotH** [reply]
That is because this has since changed when this original post was answered. Having analyst install does not take up and additional resources and is just a minor amount of additional disk space.
